python/ray/data/_internal/execution/streaming_executor_state.py

```diff
@@ -341,6 +341,10 @@ def mark_finished(self, exception: Optional[Exception] = None):
         else:
             self._exception = exception
 
+    def update_resource_usage(self) -> None:
+        """Updates resources usage."""
+        self.op.update_resource_usage()
+
```

Comment:
Where is `OpState.update_resource_usage` called?

Reply:
This is invoked by _scheduling_loop_step in streaming_executor.py. Will add a comment here.

Reply:
I thought that calls `PhysicalOperator.update_resource_usage` and not `OpState.update_resource_usage`? AFAIK `OpState.update_resource_usage` doesn't have any references

Reply:
Ah, nevermind, saw you removed `OpState.update_resource_usage`